iModelCore/ECPresentation/Source/Content/ContentSpecificationsHandler.cpp

```diff
@@ -160,6 +163,8 @@ static ECRelationshipConstraintClassList const* GetClassFromRelationship(ECSchem
         case RequiredRelationDirection_Forward:
             return &relationshipClass->GetTarget().GetConstraintClasses();
         }
+
+    DIAGNOSTICS_LOG(DiagnosticsCategory::Content, LOG_INFO, LOG_ERROR, Utf8PrintfString("Failed to get relationship constraints from relationship step specification: %s. Direction: %s.", relationshipStep->GetRelationshipClassName().c_str(), direction));

```

Comment:
`direction` is not a string, so `%s` is not okay for it here. Let's use `CommonToolsInternal::FormatRequiredDirectionString` to get a string for `direction`